src/macaron/config/defaults.ini

```diff
@@ -373,15 +380,22 @@ builder =
     yarn
 # Build args not defined for similar reasons to npm
```

Comment:
I think this comment is now not relevant with the new changes to the Yarn build tool:
- `build` is treated as `build_arg`
- new addition of `build_run_arg` and `deploy_run_arg`.

Reply:
See commit f0d89a961fe4380ceeb89526ff1e7cbacd680294